lib/documents/schemas/licence_transactions.json

```diff
@@ -0,0 +1,94 @@
+{
+  "pre_production": true,
+  "content_id": "b8327c0c-a90d-47b6-992b-ea226b4d3306",
+  "base_path": "/find-licences",
+  "format_name": "Find and apply for licences",
+  "name": "Find and apply for licences",
+  "description": "Find licences for business activities and other activities",
+  "filter": {
+    "format": "licence_transaction"
+  },
+  "show_summaries": true,
+  "organisations": ["af07d5a5-df63-4ddc-9383-6a666845ebe9"],
+  "document_noun": "licence",
```

Comment:
Just a note regarding how results order is determined - we can customise it by adding for example `"default_order": "title",` , see more examples here https://github.com/alphagov/specialist-publisher/search?q=default_order

Reply:
Ah interesting, didn't know that, thanks! I can't see an obvious ordering to apply so I'll leave this for now but will keep this in mind if there is a user need.